### ISSUE TITLE
Oculus 1.8.0 Incompatibility/Crash

### ISSUE THREAD

**Issue by YZQuan98** [leon-o/iris-flw-compat#185]
Hi there, it seems like oculus flywheel compat causing crash on client side whenever a shader is activated, along with 1.8.0 Oculus which is the latest version and support DH 2.2.

**Comment by Barerock** [comment]
Yes, crash at startup is the norm for me right now until this mod is disabled.
Crash report, this is with a bunch of other mods but I've made the error and crash consistent.
https://mclo.gs/iExvTsT

**Comment by AleksQuantum** [comment]
Yep, same thing with bliss shaders, have to wait for 3-4 minutes to activate shaders so it doesn't crash.
